src/main/scala/Util.scala

```diff
@@ -7,12 +7,14 @@ import freechips.rocketchip.diplomacy.{IdRange, ValName}
 import freechips.rocketchip.util.AsyncResetReg
 import freechips.rocketchip.tilelink._
 
-class ResetSync(c: Clock, lat: Int = 2) extends Module(_clock = c) {
+class ResetSync(c: Clock, lat: Int = 2) extends Module {
   val io = IO(new Bundle {
     val reset = Input(Bool())
     val reset_sync = Output(Bool())
   })
-  io.reset_sync := ShiftRegister(io.reset,lat)
+  withClock(c) {
+    io.reset_sync := ShiftRegister(io.reset,lat)
+  }
 }
```

Comment:
@colinschmidt @zhemao Do we even use this `ResetSync`? It seems redundant with things that exist in rocket-chip, which we import anyway.

Reply:
I always use `ResetCatchAndSync`, assuming these two do the same thing.